src/nvim/eval.lua

```diff
@@ -7148,6 +7148,9 @@ M.funcs = {
       		given sequence.
           limit	Maximum number of matches in {list} to be
       		returned.  Zero means no limit.
+          camelcase	Use enhanced camel case scoring making results
+		better suited for completion related to
+		programming languages. Default is v:true
```

Comment:
There should be 6 spaces before the first TAB

Reply:
thanks